frontend/src/pages/TrainingReports/components/SessionCard.js

```diff
@@ -0,0 +1,165 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { DECIMAL_BASE } from '@ttahub/common';
+import { Link } from 'react-router-dom';
+import UserContext from '../../../UserContext';
+import {
+  InProgress,
+  Closed,
+  NoStatus,
+  NotStarted,
+  Ceased,
+  Pencil,
+} from '../../../components/icons';
+import { canEditOrCreateSessionReports } from '../../../permissions';
+
+function SessionCard({
+  eventId,
+  session,
+  expanded,
+}) {
+  const { user } = useContext(UserContext);
+
+  const {
+    sessionName,
+    startDate,
+    endDate,
+    objective,
+    objectiveSupportType,
+    objectiveTopics,
+    objectiveTrainers,
+    status,
+    regionId,
+  } = session.data;
+
+  const hasEditPermissions = canEditOrCreateSessionReports(user, parseInt(regionId, DECIMAL_BASE));
+  const mapStatusToDisplay = [

```

Comment:
Sessions only have in progress or complete statuses

Reply:
done.